diff --git a/starVLA/dataloader/__init__.py b/starVLA/dataloader/__init__.py
index 0f324d1..4e3f26e 100644
--- a/starVLA/dataloader/__init__.py
+++ b/starVLA/dataloader/__init__.py
@@ -49,10 +49,10 @@ def build_dataloader(cfg, dataset_py="lerobot_datasets_oxe"): # TODO now here on
             vla_dataset,
             batch_size=cfg.datasets.vla_data.per_device_batch_size,
             collate_fn=collate_fn,
-            num_workers=16,
+            num_workers=4,
             pin_memory=True,
             persistent_workers=True,
-            prefetch_factor=4,
+            prefetch_factor=2,
             # shuffle=True
         )        
         if dist.get_rank() == 0: 
